clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ClusterController.groovy

```diff
@@ -43,6 +44,7 @@ class ClusterController {
   @Autowired
   MessageSource messageSource
 
+  @PreAuthorize("hasPermission(#application, 'APPLICATION', 'READ')")
   @RequestMapping(method = RequestMethod.GET)
   Map<String, Set<String>> list(@PathVariable String application) {
```

Comment:
The method name `list` is slightly confusing here. At first glance, looks like `list` should be account scoped as well, but it actually just gives you the cluster names, correct? I might rename this function to be more precise - `listClustersByName` or something appropriate?